src/main/java/xyz/aadev/generitech/client/gui/button/ButtonUpgrade.java

```diff
@@ -17,25 +17,29 @@
  * Exclusive Remedies. The Software is being offered to you free of any charge. You agree that you have no remedy against AlgorithmicsAnonymous, its affiliates, contractors, suppliers, and agents for loss or damage caused by any defect or failure in the Software regardless of the form of action, whether in contract, tort, includinegligence, strict liability or otherwise, with regard to the Software. Copyright and other proprietary matters will be governed by United States laws and international treaties. IN ANY CASE, AlgorithmicsAnonymous SHALL NOT BE LIABLE FOR LOSS OF DATA, LOSS OF PROFITS, LOST SAVINGS, SPECIAL, INCIDENTAL, CONSEQUENTIAL, INDIRECT OR OTHER SIMILAR DAMAGES ARISING FROM BREACH OF WARRANTY, BREACH OF CONTRACT, NEGLIGENCE, OR OTHER LEGAL THEORY EVEN IF AlgorithmicsAnonymous OR ITS AGENT HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES, OR FOR ANY CLAIM BY ANY OTHER PARTY. Some jurisdictions do not allow the exclusion or limitation of incidental or consequential damages, so the above limitation or exclusion may not apply to you.
  */
 
-import net.minecraft.entity.player.InventoryPlayer;
-import xyz.aadev.aalib.client.gui.GuiBase;
-import xyz.aadev.generitech.Reference;
-import xyz.aadev.generitech.common.container.power.ContanierGenerator;
-import xyz.aadev.generitech.common.tileentities.power.TileEntityPowerStorage;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiButton;
+
+public class ButtonUpgrade extends GuiButton {
+    public ButtonUpgrade(int buttonId, int x, int y) {
+        super(buttonId, x, y, "");
+        this.visible=true;
+        this.width = 20;
+        this.height = 20;
 
-public class GuiPowerStorage extends GuiBase {
 
-    public GuiPowerStorage(InventoryPlayer inventoryPlayer, TileEntityPowerStorage tileEntity) {
-        super(Reference.MOD_ID, new ContanierGenerator(inventoryPlayer, tileEntity));
-    }
 
+    }
     @Override
-    public void drawBG(int paramInt1, int paramInt2, int paramInt3, int paramInt4) {
-// Do nothing because of not implemented
+    public void drawButton(Minecraft mc, int mouseX, int mouseY) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1185)